File: harvey/2920.py
import sys

read = lambda : sys.stdin.readline().strip()

l = list(map(int, read().split()))
p = ''

for i in range(1, len(l)):
    if i/l[i-1] == 1 and p !='descending':
        p = 'ascending'
    elif i+l[i-1] == 9 and p !='ascending':
        p = 'descending'
    else:
        print('mixed')
        exit()

print(p)

File: chu/solved/2920.js
var fs = require('fs');

var ascending = [1,2,3,4,5,6,7,8];
var descending = [8,7,6,5,4,3,2,1];
var sum = 0;

var input = fs
            .readFileSync('../stdin/2920')
            .toString().split(' ')
            .map(function(e){
                return parseInt(e);
            });

for( index in input ){
    if(input[index] === ascending[index])
        sum++;
    if(input[index] === descending[index])
        sum--;
}

if( sum === 8 )
    console.log("ascending");
else if( sum === -8 )
    console.log("descending");
else 
    console.log("mixed");